File: app/src/main/java/net/bonono/rssreader/RssReaderApplication.java
package net.bonono.rssreader;

import android.app.Application;

import com.jakewharton.threetenabp.AndroidThreeTen;

import io.realm.Realm;

public class RssReaderApplication extends Application {
    @Override
    public void onCreate() {
        super.onCreate();
        AndroidThreeTen.init(this);
        Realm.init(this);
    }
}

File: app/src/main/java/net/bonono/rssreader/RssReaderApplication.kt
package net.bonono.rssreader

import android.app.Application

import com.jakewharton.threetenabp.AndroidThreeTen

import io.realm.Realm

class RssReaderApplication : Application() {
    override fun onCreate() {
        super.onCreate()

        AndroidThreeTen.init(this)
        Realm.init(this)
    }
}
